File: Lv1/크기가작은부분문자열.java
package Lv1;

public class 크기가작은부분문자열 {

    public int solution(String t, String p) {

        int answer = 0;
        int length = p.length();

        for(int i = 0; i + length <= t.length(); i++) {
            if(Long.parseLong(t.substring(i, i + length)) <= Long.parseLong(p)) {
                answer++;
            }
        }

        return answer;
    }

}

File: kotlin/Lv1/크기가작은부분문자열.kt
package kotlin.Lv1

class 크기가작은부분문자열 {

    fun solution(t: String, p: String): Int {
        var answer: Int = 0
        val pLength = p.length
        val tLength = t.length

        for (i in 0..tLength - pLength) {
            if (t.substring(i, i + pLength).toLong() <= p.toLong()) {
                answer++
            }
        }
        return answer
    }

}